src/compressed_tensors/transform/factory/matrix_multiply.py

```diff
@@ -74,17 +74,27 @@ def _create_inverse(self, weight: Parameter) -> Parameter:
 
 
 class RandomMatrixTransform(TransformBase):
-    def __init__(self, weight: Tensor, args: TransformArgs):
+    def __init__(
+        self,
+        weight: Tensor,
+        args: TransformArgs,
+        module_type: type[torch.nn.Module],
```

Comment:
I don't think this will work on python 3.9? I believe you have to use `typing.List/Dict/Type` instead of `list/dict/type` built-ins for typehints

Reply:
typing.Type has actually be depreciated since 3.9
https://docs.python.org/3/library/typing.html#typing.Type

Reply:
oh, can we use `list` instead of `typing.List` then? I thought we weren't because we were supporting 3.9 still
https://docs.python.org/3/library/typing.html#typing.List

Reply:
just checked, these seem to be working in python 3.9:

```python
>>> def f(a: type, b: list):
...     pass
... 
>>> f(type(int, []))
>>> def g(a: type[int], b: list[float]):
...     pass
... 
>>> g(type(4), [1.2])
```

Reply:
I'd prefer to use `list` primatives as well